documentation/README.md

```diff
@@ -3,8 +3,10 @@ This is the Rogue API, it is used to capture activity from members.
 All `POST` and `PUT` endpoints require an api key (`X-DS-Rogue-API-Key`) in the header to be submitted with the request. 
 
 ## Endpoints
+
+### Web
 #### Reviews
-Endpoint                                       | Functionality                                           
```

Comment:
Totally not important but looks like formatting got a little borked without the `|` here:

![screen shot 2017-11-30 at 11 34 06 am](https://user-images.githubusercontent.com/583202/33442137-64622b14-d5c2-11e7-8088-cefb9d4fb644.png)